src/Presentation/FacetUiBuilder.php

```diff
@@ -54,23 +56,26 @@ private function facetsToViewModel( /* TODO: parameters */ ): array {
 				'values-html' => $this->getItemsHtml(
 					$this->getExampleListItems(), FacetType::LIST->value, 'Author'
 				),
-				'expanded' => true
+				'expanded' => true,
+				'hasToggle' => true
```

Comment:
Not a technical issue, but `hasToggle` and `FacetToggle` seem a bit generic. Although we don't have any other toggles at the moment.

Reply:
I agree.
The attribute is currently a placeholder and should be shortly swap out since we only use it for list facet.
The value/config should be derived from the facet type instead, when we have more solid spec on the facet config.